packages/taiko-client/driver/preconf_server/server.go

```diff
@@ -75,6 +75,6 @@ func (s *PreconfAPIServer) configureMiddleware(corsOrigins []string) {
 func (s *PreconfAPIServer) configureRoutes() {
 	s.echo.GET("/", s.HealthCheck)
 	s.echo.GET("/healthz", s.HealthCheck)
-	s.echo.POST("/perconfTransactions", s.CreateOrUpdateBlocksFromBatch)
+	s.echo.POST("/preconfTransactions", s.CreateOrUpdateBlocksFromBatch)
```

Comment:
I think we should use "preconfirm_transactions" or "preconfirmTransactions", and "remove_tentative_blocks"

Reply:
Changed